xml/System.Text.Json/JsonReaderException.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Defines a custom exception object that is thrown by the <see cref="T:System.Text.Json.Utf8JsonReader" /> whenever it encounters an invalid JSON text while reading through it. This exception is also thrown whenever you read past the defined maximum depth.</summary>
```

Comment:
Based on https://docs.microsoft.com/en-us/dotnet/api/system.io.ioexception?view=netframework-4.7.2 I'd expect something more like

`The exception that is thrown when <see cref="T:System.Text.Json.Utf8JsonReader" /> encounters invalid data, or data disallowed by any of its options.`

Nice and general, so it doesn't need to be updated over time.  (I'd even sort of suggest being "a JSON reader" instead of the specific type, but we can address that when/if we have a second one)